modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/GodotMethodAttribute.cs

```diff
@@ -2,13 +2,20 @@
 
 namespace Godot
 {
+    /// <summary>
+    /// An attribute for a method.
+    /// </summary>
     [AttributeUsage(AttributeTargets.Method)]
     internal class GodotMethodAttribute : Attribute
     {
         private string methodName;
 
         public string MethodName { get { return methodName; } }
 
+        /// <summary>
+        /// Attribute constructor.
```

Comment:
Look through your changes and do a similar thing to the rest of the constructors.